docs/neomutt.man

```diff
@@ -189,6 +189,21 @@ Run in batch mode (do not start the ncurses UI)
 Specify a blind carbon copy (Bcc) recipient
 .
 .TP
+.BI \-C
+Enable cryptographic operations in the cases in which they're disabled by
+default. Those include:
```

Comment:
As maintainer of the Linux man-pages, I must point out the following ;-)

```
$ MANWIDTH=72 man 7 man-pages | sed -n '/Use semantic newlines/,/^$/p';
   Use semantic newlines
     In  the  source of a manual page, new sentences should be started
     on new lines, long sentences should be split into lines at clause
     breaks (commas, semicolons, colons, and so on), and long  clauses
     should be split at phrase boundaries.  This convention, sometimes
     known  as  "semantic newlines", makes it easier to see the effect
     of patches, which often operate at the level of  individual  sen‐
     tences, clauses, or phrases.

```

and more details here:
<https://www.alejandro-colomar.es/src/alx/linux/man-pages/man-pages.git/commit/man7/man-pages.7?id=6ff6f43d68164f99a8c3fb66f4525d145571310c>

Reply:
Ah, sorry!
Line break restored.

Reply:
BTW, I noticed a problem regarding spaces after period in the manual page.  The right amount in source code is 2, no matter what style guides say these days.  :)

I'll open an issue and write some patch.